introduction.md

```diff
@@ -6,3 +6,10 @@ Arigato gozaimasu~
 Thank you for accepting me into this dojo. I am excited to learn.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?